WestLaw UK.js

```diff
@@ -73,10 +73,11 @@ function scrapeCase(doc, url) {
 	translator.setHandler('itemDone', function (obj, item) {
 		const citationArray = parseCitationList(text(doc, "#co_docContentWhereReported", 0));
 
+		item.title = text(doc, "#co_docHeaderContainer");
```

Comment:
is it worth checking whether this exists before overwriting the EM title, to make this more robust? (and otherwise just remove the addition from the existing item title)?

Reply:
All added